### PR TITLE
docs: fix article "Add a Subnet" to guide the user to click Resource Name

### PR DESCRIPTION
**Pull request by dudunog**
## Description of changes

- [ ]  Fixed the excerpt "In the list of networks, find the network you want to manage and click DETAILS" that was replaced by a more descriptive text explaining how to enter in the subnet details view.
- [ ]  Fixed the excerpt "Follow the prompts then click CONFIRM" that was replaced by an explanation about every step to add a subnet, like adding a name, selecting a subnet type and a subnet zone.

## GitHub issues resolved by this PR

- This PR solves #881 

## Quality Assurance

- Once the changes in this PR are merged and deployed, success criteria is:  Update the article https://www.devopness.com/docs/networks/add-network/

### CODE REVIEW

File: docs/subnets/add-subnet.md [hunk 1]
```diff
@@ -16,8 +16,15 @@ required_permissions:
 1. On Devopness, navigate to a project then select an environment
 1. Find the `Networks` card
 1. Click `View` in the `Networks` card, to see a list of existing `Networks`
-1. In the list of networks, find the network you want to manage and click `DETAILS`
+1. In the list of networks find the network where you want to add a subnet and click the `NAME` of the network
 1. On the upper-right corner of network details view, click `SUBNETS`
 1. On the upper-right corner of subnets list, click `ADD SUBNET`
-1. Follow the prompts then click `CONFIRM`
+1. Click `NEXT`
+1. Provide a name to the Network being added. Example: "Automate Net Subnet"
```
Comment (jfoliveira) [inline docs/subnets/add-subnet.md:23]: Perhaps give simpler examples for the subnet name?

```suggestion
1. Provide a name to the Subnet being added. Example: "Public Subnet A", "Private Subnet B"
```

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/subnets/add-subnet.md (hunk 1 above)
Comment (dudunog) [reply docs/subnets/add-subnet.md]: Great, fixed!

**Review by thlmenezes — APPROVED** [review]

**Comment by thlmenezes** [comment]
@jfoliveira could you review this PR again?

File: docs/subnets/add-subnet.md [hunk 2]
```diff
@@ -16,8 +16,15 @@ required_permissions:
...
-1. In the list of networks, find the network you want to manage and click `DETAILS`
+1. In the list of networks find the network where you want to add a subnet and click the `NAME` of the network
 1. On the upper-right corner of network details view, click `SUBNETS`
 1. On the upper-right corner of subnets list, click `ADD SUBNET`
-1. Follow the prompts then click `CONFIRM`
+1. Click `NEXT`
+1. Provide a name to the Subnet being added. Example: "Public Subnet A", "Private Subnet B"
+1. Select a `Type`
+1. Click `NEXT`
+1. Select a `ZONE` and click `NEXT`
```
Comment (jfoliveira) [inline docs/subnets/add-subnet.md:26]: Some fields in the `Add subnet` form are cloud provider specific.
For example: on `AWS` a `Zone` is required to create a subnet, whereas on `GCP` a `Zone` is not required but a `Region` is required to create a subnet.
In cases like this perhaps makes more sense to use a generic text such as 
```
Follow the prompts then click ...
```
as used in the docs for [Add a Network](https://www.devopness.com/docs/networks/add-network/)?

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/subnets/add-subnet.md [hunk 3]
```diff
@@ -16,8 +16,15 @@ required_permissions:
 1. On Devopness, navigate to a project then select an environment
```
Comment (jfoliveira) [inline docs/subnets/add-subnet.md:16]: @dudunog could you please update the PR description linking the PR with the issue(s) it's closing?
For details, see [GitHub syntax for linking issues to PRs](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword#linking-a-pull-request-to-an-issue-using-a-keyword)

**Review by jfoliveira — CHANGES_REQUESTED** [review]

File: docs/subnets/add-subnet.md (hunk 2 above)
Comment (dudunog) [reply docs/subnets/add-subnet.md]: Ok, fixed.

**Review by jfoliveira — APPROVED** [review]

**Pull request merged**